src/Mailer.hs

```diff
@@ -132,4 +140,5 @@ mailer genEmail settings = reactivelyRunAction
         -- email to an invalid address. At the moment reactivelyRunAction will
         -- just catch/hide that from us, and because we're just reacting to
         -- current changes, we'll never repeat the email attempt!
-        sendEmails settings emails >> return (Emailed <$> pending)
+        -- FIXME: really want to be operating in the space of email events only:
```

Comment:
Is this just that doing the decompose at the moment would be irritating because the other 2 handlers would be no-op?